Beatles.sol

```diff
@@ -34,7 +34,7 @@ contract Beatles is
     uint256 public constant MAX_QTY_PER_MINTER_IN_PRESALES = 2;
 
     // max sales quantity
-    uint256 public constant SALES_MAX_QTY = TOTAL_MAX_QTY - GIFT_MAX_QTY;
+    uint256 public constant SALES_MAX_QTY = 9000;
```

Comment:
I heard community decided not to limit its supply , only use 24 hours to guard how many can be minted.
So here may not be a good idea to hard code this value.

Reply:
Unless there is a set method for the Gift Max, doing calculation every single mint is waste of gas.